languages.json

```diff
@@ -3,7 +3,7 @@
 		"Name": "Bash",
 		"Language": "bash",
 		"Status": "beta",
-		"SDKVersion": "2.16.0",
+		"SDKVersion": "2.16.4",
```

Comment:
Oops, I guess we forgot to do this the last couple times. 😢

Reply:
It's quite hard to update on each point release of SDK without automation, so we usually update only if new release improves the performance or fixes something for this driver.

Reply:
Agreed—I wasn't complaining, but it's a good point in favour of finding a way to automate the check, at least (as we've discussed previously).